app/src/main/java/io/multy/viewmodels/WalletViewModel.java

```diff
@@ -100,14 +103,19 @@ public WalletRealmObject createWallet(String walletName) {
             if (!Prefs.getBoolean(Constants.PREF_APP_INITIALIZED)) {
                 Multy.makeInitialized();
                 FirstLaunchHelper.setCredentials("");
+                ServerConfigResponse serverConfig = EventBus.getDefault().removeStickyEvent(ServerConfigResponse.class);
+                if (serverConfig != null) {
+                    RealmManager.open(Multy.getContext());
+                    RealmManager.getSettingsDao().saveDonation(serverConfig.getDonate());
+                }
             }
             DataManager dataManager = DataManager.getInstance();
 
             final int topIndex = Prefs.getInt(Constants.PREF_WALLET_TOP_INDEX, 0);
 
-            if (!Prefs.getBoolean(Constants.PREF_APP_INITIALIZED)) {
-                FirstLaunchHelper.setCredentials("");
-            }
+//            if (!Prefs.getBoolean(Constants.PREF_APP_INITIALIZED)) {
```

Comment:
why this is uncommented?

Reply:
uncommented?
I was looking in this class and saw the same lines in 103 line and 116. Is it really must being uncommented?